setup.cfg

```diff
@@ -30,6 +30,8 @@ long_description = Meson is a cross-platform build system designed to be both as
 
 [options]
 python_requires = >= 3.5.2
+setup_requires =
```

Comment:
If you use it at runtime, shouldn't it be `install_requires`?

I would imagine having `setuptools` in `setup_requires` would not have any effect since setuptools is itself what interprets `setup_requires`, though I'm not 100% sure.

Reply:
Normally I would agree with you, but I think debian splits pkg_resource out of setuptools.